packages/cli/lib/imageslice.js

```diff
@@ -0,0 +1,255 @@
+        iiifConfig
+      )} -quality default -verbose -refresh -scale-factors 128,64,32,16,8,4,2,1,0 ${imageName}`;
+
+      return new Promise(function (resolve, reject) {
+        // execSync(cmd); // alternative command, more console logs
+        return execa.commandSync(cmd) ? resolve(true) : reject(true);
+      });
+    }
+
+    // Get all images in directory recursively, if it happens to detect any files
+    // with 'color', it'll rename them to 'default' with the appopriate extension
+    async function getAllImagesAndRename(dir) {
+      const entryPaths = fs
+        .readdirSync(dir)
+        .map((entry) => path.join(dir, entry));
+      const filePaths = entryPaths.filter((entryPath) =>
+        fs.statSync(entryPath).isFile()
+      );
+      const dirPaths = entryPaths.filter(
+        (entryPath) => !filePaths.includes(entryPath)
+      );
+      const dirFiles = dirPaths.reduce(
+        (prev, curr) => prev.concat(getAllImagesAndRename(curr)),
+        []
+      );
+
+      if (filePaths.length && filePaths[0].includes("color")) {
+        const file = filePaths[0];
+        const dir = file.substr(0, file.lastIndexOf("/"));
+        const newFile = dir + "/default.jpg";
+        const fileAttrs = filePaths[1];
+        const newAttrsFile = dir + "/default.jpg.attrs";
+        fs.renameSync(file, newFile);
+        fs.renameSync(fileAttrs, newAttrsFile);
+      }
+    }
+
+    // check if directories exist and find all images;
+    // also check if any images have been processed and 
+    // delete them so it can do a fresh slice
+    function getAllImages() {
+      spinner.info("Getting all images");
+      if (fs.existsSync(iiifSeed)) {
+        let files = fs.readdirSync(iiifSeed);
+        for (let i = 0, len = files.length; i < len; i++) {
+          let fullPath = path.join(iiifSeed, files[i]);
+          let stat = fs.lstatSync(fullPath);
+          let fullProcessPath = path.join(iiifProcessed, files[i]);
+
+          const params = [".jpg", ".jpeg", ".png", ".svg", ".jp2"];
+          if (stat.isDirectory()) {
+            spinner.fail(
+              `Directories found! Please delete and move images you'd like to slice to ${iiifSeed}`
+            );
+            throw new error();
+          } else {
+            if (params.some((el) => fullPath.includes(el))) {
+              images.push(fullPath);
+            } else {
+              spinner.fail(`No images found! Add images in: ${iiifSeed}`);
+              throw new error();
+            }
+          }
+          if (fs.existsSync(fullProcessPath)) {
+            let statProcessed = fs.lstatSync(fullProcessPath);
+            if (statProcessed.isDirectory()) {
+              spinner.info(
+                "Processed image found; cleaning up processed image directory"
+              );
+              // execa.commandSync(`rm -rf ${statProcessed}`);
+              rimraf.sync(fullProcessPath);
```

Comment:
Above, suggested that `fullProcessedPath` be renamed to `outputPath`
```suggestion
              rimraf.sync(outputPath);
```